pkg/cmd/controller/route/interfaces.go

```diff
@@ -54,8 +52,7 @@ type ControllerContext struct {
 	RouteInformers    routeinformer.SharedInformerFactory
 	OperatorInformers operatorinformer.SharedInformerFactory
 
-	GenericResourceInformer genericinformers.GenericResourceInformer
```

Comment:
Just delete this instance and everything in this file using it.  The other non-route controllers need it.

Reply:
seems like it moved all away, to the cluster-policy-controller